runbotics/runbotics-desktop/.env.development

```diff
@@ -1,3 +1,6 @@
+ENCRYPTED=false
+ENC_KEY=boticsRunboTics$
```

Comment:
no need to have 2 ENVs. `ENC_KEY` existence will tell us whether decrypt those values or not